src/Tabs/Tabs.component.tsx

```diff
@@ -0,0 +1,169 @@
+// VENDOR
+import * as React from 'react';
+import classNames from 'classnames';
+import styled, { css } from '@xstyled/styled-components';
+import { th, space as spaceStyles, SpaceProps } from '@xstyled/system';
+// ANCHOR
+import { cloneWithProps } from '../utils/cloneWithProps/cloneWithProps';
+import { Position, Variant, variantStyles } from './utils';
```

Comment:
Really cool job composing these variants. Did you come up with this or borrow it?

Reply:
I came up with it. Haven't seen it anywhere else